### ISSUE TITLE
Covariate data take too long to download?

### ISSUE THREAD

**Issue by aknandi** [aknandi/TMBworkshop#10]
Covariate data is taken from WorldClim using getData at 5 minutes resolution. This took be about 2 minutes to download. Probably ok but something to think about if there is bad internet, etc.
(10 minutes resolution is very coarse)

**Comment by timcdlucas** [comment]
Yeah, I don't think we have many options to a) make data available on online sources and b) make download very quick.

Maybe, just as soon as the talk starts get people to run the first few lines and leave it downloading in the background or something.

**Comment by timcdlucas** [comment]
I think I've dealt with this in a sensible way.

Firstly, I've separated the incidence data download script. We want to avoid rerunning prepare_data.R everytime as it downloads the PR data which is kinda slow. Then commented out (but left in) the `source('prepare_data.R')` in the model 2 and 3.

Then I've moved the compile and load bits to near the model. This way as soon as people sit down we can ask them to run the first 10 lines (libs and prepare data) so the covariates will download while everyone is filtering in etc.
